### PR TITLE
fix: onerror mem leak

### LINKED ISSUE #639: onerror memory leak
- Operating System: Mac
- Node Version: 12
- NPM Version: 12.18.2
- webpack Version: 4.44.2
- mini-css-extract-plugin Version: 1.2.0
- browser:  chrome 86.0.4240.111

### Expected Behavior

should unref onerror after onload

### Actual Behavior

onerror remains in the memory

截屏2020-10-26 下午7 25 22

### Code

https://github.com/dgeibi/mini-css-extract-plugin-mem-leak-demo

### How Do We Reproduce?

```
git clone https://github.com/dgeibi/mini-css-extract-plugin-mem-leak-demo
cd mini-css-extract-plugin-mem-leak-demo
npm i
npm run build
# open dist/index.html with chrome
```

### PR DESCRIPTION
**Pull request by dgeibi**
This PR contains a:

- [x] **bugfix**
- [ ] new **feature**
- [ ] **code refactor**
- [x] **test update** 
- [ ] **typo fix**
- [ ] **metadata update**

### Motivation / Use-Case

Fixes #639 

### Breaking Changes

No

### Additional Info

No

### CODE REVIEW

**Review by evilebottnawi — COMMENTED** [review]
Example of fix https://github.com/webpack/webpack/blob/db36afea4dfe0b76e66c04cd9429194c3b140002/lib/runtime/LoadScriptRuntimeModule.js#L125

**Comment by dgeibi** [comment]
@evilebottnawi  Do you mean I should add this line  "prev(event)" ?

**Comment by evilebottnawi** [comment]
We should move onerror handler to separate function and handle this (look at link), we should generate runtime as little as possible

**Comment by dgeibi** [comment]
Is the generated code ok like this?
```js
var onStyleComplete = function (event) {
  // avoid mem leaks
  linkTag.onerror = linkTag.onload = null;
  if (event.type === 'load') {
    resolve();
  } else {
    var request = (event && event.target && event.target.href) || fullhref;
    var err = new Error(
      'Loading CSS chunk ' + chunkId + ' failed.\n(' + request + ')'
    );
    err.code = 'CSS_CHUNK_LOAD_FAILED';
    err.request = request;
    linkTag.parentNode.removeChild(linkTag);
    reject(err);
  }
};
linkTag.onerror = linkTag.onload = onScriptComplete;
```

**Comment by evilebottnawi** [comment]
Yes :+1:

File: src/CssLoadingRuntimeModule.js [hunk 1]
```diff
@@ -60,17 +60,25 @@ module.exports = class CssLoadingRuntimeModule extends RuntimeModule {
...
+            '} else {',
+            Template.indent([
+              'var request = event && event.target && event.target.href || fullhref;',
+              'var err = new Error("Loading CSS chunk " + chunkId + " failed.\\n(" + request + ")");',
+              'err.code = "CSS_CHUNK_LOAD_FAILED";',
+              'err.request = request;',
+              'linkTag.parentNode.removeChild(linkTag)',
+              'reject(err);',
+            ]),
+            '}',
```
Comment (evilebottnawi) [inline src/CssLoadingRuntimeModule.js:78]: Please look at code provided by my :disappointed:

**Review by evilebottnawi — APPROVED** [review]
Good job, thanks

**Pull request merged**